### PR TITLE
data: rewrite provisionActivity field

### PR DESCRIPTION
**Pull request by rerowep**
* Rewrits rero-ils, ebooks and unimarc dojson.
* Updates fixtures.
* Updates tests.
* Updates json schema.
* Updates json mapping.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- https://tree.taiga.io/project/rero21-reroils/us/1090?milestone=250540

## How to test?

- start `setup` and find records with `revisionActivity`

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

**Comment by jma** [comment]
Can you fix your commit message?

File: tests/data/data.json [hunk 1]
```diff
@@ -1075,10 +1075,17 @@
...
             ],
             "type": "bf:Agent"
+          },
+          {
+            "label": [
+              {
+                "value": "1971"
+              }
+            ],
+            "type": "bf:Agent"
```
Comment (jma) [inline tests/data/data.json]: date?

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json [hunk 2]
```diff
@@ -1169,8 +1169,33 @@
               "bf:Production"
             ]
           },
+          "place": {
```
Comment (jma) [inline rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json]: can you add a minItems properties at each time you have an array?

File: tests/data/data.json (hunk 1 above)
Comment (rerowep) [reply tests/data/data.json]: Yup

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json (hunk 2 above)
Comment (rerowep) [reply rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json]: ok

File: rero_ils/modules/documents/dojson/contrib/marc21tojson/model.py [hunk 3]
```diff
@@ -392,8 +391,31 @@ def build_place_or_agent_data(code, label, index, link, add_country):
             publication['endDate'] = end_date
         if (marc21tojson.date_type_from_008 == 'q' or
                 marc21tojson.date_type_from_008 == 'n'):
-            publication['note'] = 'Date(s) incertaine(s) ou inconnue(s)'
+            publication['note'] = _('Date(s) incertaine(s) ou inconnue(s)')
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/documents/dojson/contrib/marc21tojson/model.py]: ok

**Pull request merged**